CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the ld-find-code-refs program will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org).
 
+## [0.4.1] - 2019-01-31
```

Comment:
(nitpick) Should this be 0.5.0? https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
oh yeah forgot about this. i'm gonna remove the version tag actually and add it back in with the db/rpm PR